plugin/trino-thrift/src/test/java/io/trino/plugin/thrift/integration/TestThriftConnectorTest.java

```diff
@@ -23,9 +23,8 @@
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static io.trino.testing.QueryAssertions.assertContains;
 
-public class TestThriftIntegrationSmokeTest
-        // TODO extend BaseConnectorTest
-        extends AbstractTestIntegrationSmokeTest
+public class TestThriftConnectorTest
```

Comment:
There probably needs to be provided an own implementation of `io.trino.testing.BaseConnectorTest#hasBehavior` method here due to the fact that this connector does not support all the operations tested in `BaseConnectorTest`.
Use `io.trino.plugin.cassandra.TestCassandraConnectorTest#hasBehavior` as a template.


Use `io.trino.plugin.thrift.ThriftMetadata` and `io.trino.spi.connector.ConnectorMetadata` to see what is possible in this connector.

Reply:
> There probably needs to be provided an own implementation of io.trino.testing.BaseConnectorTest#hasBehavior method here due to the fact that this connector does not support all the operations tested in BaseConnectorTest.
Use io.trino.plugin.cassandra.TestCassandraConnectorTest#hasBehavior as a template.

Thank you for the reviewed!  I will do it。

>Use io.trino.plugin.thrift.ThriftMetadata and io.trino.spi.connector.ConnectorMetadata to see what is possible in this connector.

Sorry，I don't understand it。Do you mean to test the functionality of ThriftMetadata?

Reply:
> Sorry，I don't understand it。Do you mean to test the functionality of ThriftMetadata?

 By checking the code of the previously mentioned classes, you'll be able to figure out what values of the `io.trino.testing.TestingConnectorBehavior` you can add to the `hasBehavior` method .

Note that these values control whether a test needs to be executed or not
```
        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE));
```